src/services/formatting/smartIndenter.ts

```diff
@@ -322,6 +322,26 @@ namespace ts.formatting {
             return false;
         }
 
+        export function childStartsInlineWithPreviousObject(parent: Node, child: TextRangeWithKind, childStartLine: number, sourceFile: SourceFileLike): boolean {
+            if (parent.kind === SyntaxKind.CallExpression) {

```

Comment:
Should probably check for `NewExpression` too—you can use `isCallOrNewExpression(parent)` which will also eliminate the need for the type assertion below.